components/06-examples/beheer-basis.hbs

```diff
@@ -0,0 +1,169 @@
+{{> '@header' }}
+<main>
+  <div class="clearfix">
```

Comment:
Vanwaar de `.clearfix` ? Clearfix is niet op deze manier te gebruiken: https://getbootstrap.com/docs/3.4/css/#helper-classes-clearfix

Reply:
Ik had als tip gekregen om `.clearfix` als (tijdelijke) workaround te gebruiken als er teveel witruimte wordt gerenderd. Zie screenshots. 

Wat moet ik in de code aanpassen zodat de witruimte goed wordt gerenderd, zonder dat hier `.clearfix` voor gebruikt wordt?

![Screenshot - zonder clearfix](https://user-images.githubusercontent.com/4227175/71900672-8964e880-315e-11ea-8fbc-9997ae1db531.png)

![Screenshot - met clearfix workaround](https://user-images.githubusercontent.com/4227175/71900687-91bd2380-315e-11ea-95e5-781ff0fa1193.png)

Reply:
De befaamde witruimtes tussen rows :-) Ooit leek het een goed idee.. Maar wellicht dat @bramhoosemans, @MichielDHVD en ik dan toch de conclusie moeten trekken dat het er in z'n geheel uit moet.

De oorsprong is in ieder geval te vinden in #61 

De regel is: `main > .row` krijgt de verantwoordelijkheid om visueel secties aan te brengen. Zo kan een row groen gemaakt worden, een border krijgen, etc. Deze row zorgt ook voor de ruimte tussen de inhoud en header/footer.

Maar wat denken jullie van de volgende oplossing: We styling gaat niet meer op `main > .row` maar op `main > section.row, main > article.row` en verwacht de volgende markup:

```html
<main>
  <section class="row">
    <div class="col-sm-6">inhoud</div>
    <div class="col-sm-6">meer inhoud</div>
  </section>
  <article class="row">
    [..]
  </article>
</main>
```

Als iets visueel een eigen sectie moet worden, rechtvaardigd het dat dan ook niet in de markup? Mocht visueel geen eigen sectie zijn maar alles "achter elkaar", volstaat een div-element.

@gofix Hoe denk jij hierover?

Overigens is `.clearfix` dus een slechte workaround. De `.clearfix` wordt _na_ gefloat elementen ingezet en is zeker niet gemaakt om kind elementen te bevatten.

Als je dit al opgelost wilt hebben, gebruik dan een lege div

Reply:
Tnx. Ik hanteer voor nu dan een lege `<div>` en niet meer `.clearfix`. 

Verder, ik herinner mij iets over het gebruik van `<section>` met screen readers. Bijgaand een afbeelding en ook de beschrijvende tekst: 
<img width="837" alt="Mixed support voor Section bij screen readers" src="https://user-images.githubusercontent.com/4227175/72812479-dcac6000-3c61-11ea-86a9-3e9920bb256e.png">

`<section>`: 
"Mixed support: Most screen readers list it only if given a name via aria-label or aria-labelledby.
Note that if each section has a heading, screen reader users will be able to navigate by headings, so it is usually not necessary to give each section or region a name, and in fact it can be a bad idea to supply too many landmarks in a page, because it slows down the user's ability to sort through them all."

Reply:
Nog even terugkomen op mijn opmerking dat clearfix verkeerd werd ingezet: Het is zeker een (slechte) hack om de witruimtes te voorkomen, maar wat ik zei dat het geen kind elementen mag bevatten is niet waar. Ik kwam de clearfix op dezelfde manier ergens anders tegen en heb nogmaals de Bootstrap documentatie bekeken, en het is helemaal prima om het `clearfix` element als container te gebruiken:

* https://getbootstrap.com/docs/3.3/css/#less-mixins-clearfix
* https://getbootstrap.com/docs/3.4/css/#helper-classes-clearfix

Ik heb alleen het volgende voorbeeld verwerkt: https://getbootstrap.com/docs/3.4/css/#grid-responsive-resets Maar er is dus meer mogelijk.

My bad! Bij deze gerectificeerd